enterprise/frontend/src/metabase-enterprise/ai-entity-analysis/utils.ts

```diff
@@ -1,20 +1,34 @@
 import type { VisualizationDisplay } from "metabase-types/api";
 
+const CHART_ANALYSIS_ENABLED: { [type in VisualizationDisplay]: boolean } = {
```

Comment:
I changed the approach so that this will type error if someone adds a new chart type.

Reply:
Smart 👍

Reply:
Good catch. I had made our query for the SVG DOM node too restrictive for some charts. I've corrected this and now it should work as expected.